src/accelerate/commands/launch.py

```diff
@@ -812,7 +813,9 @@ def _validate_launch_command(args):
             and not args.use_megatron_lm
         ):
             args.use_deepspeed = defaults.distributed_type == DistributedType.DEEPSPEED
-            args.multi_gpu = defaults.distributed_type == DistributedType.MULTI_GPU
+            args.multi_gpu = (
+                True if defaults.distributed_type in (DistributedType.MULTI_GPU, DistributedType.MULTI_XPU) else False
+            )
```

Comment:
```suggestion
            args.multi_gpu = defaults.distributed_type in (DistributedType.MULTI_GPU, DistributedType.MULTI_XPU)
```

We can clean this up a bit to keep in line with the rest of the logic